src/utils.h

```diff
@@ -94,7 +95,7 @@ namespace ledger {
 
 #else // !NO_ASSERTS
 
-#define assert(x)
+#define assert(x) ((void)(x))
```

Comment:
The Python API uses the assert macro in several places (e.g. [`main/Objects/unicodeobject.c:118`](https://github.com/python/cpython/blob/main/Objects/unicodeobject.c#L118).

I'm uncertain whether this change is the best change possible, yet it allows the Ledger to compile successfully.